source/dea_shutdown.rst

```diff
@@ -3,11 +3,20 @@
 DEA-A Anomalous Shutdown
 ========================
 
+.. note::
+

```

Comment:
This note is not appearing correctly because the text after `.. note::` needs to be indented to the same spacing as the "note" word is. 

It also might be worth using `.. warning::` instead here.